frontend/src/views/web/auth/signup.html

```diff
@@ -30,9 +30,12 @@
         </div>
         <!-- password -->
         <div class="input-field align-left">
-            <input type="password" id="password" onpaste="return false;" class="dark-autofill" name="password" ng-model="auth.regUser.password" ng-minlength="8" required>
+            <input type="password" id="password" onpaste="return false;" class="dark-autofill" name="password" ng-model="auth.regUser.password" ng-minlength="8" ng-change= "auth.checkStrength(password)" required>
             <span class="form-icon form-icon-dark"><i class="fa fa-key"></i></span>
-            <label for="password">Password (min 8 characters) *</label>
+            <label for="password">
+                <strong>Password(minlength:8,required:*[a-z],*[A-Z],*[0-9],*[!@#\$%\^&\*])</strong>
```

Comment:
Replace this with Password(minlength:8,required:*[a-z],*[A-Z],*[0-9],*[!@#\$%\^&\*]) with Password - minimum length:8 required.

Reply:
@aka-jain it is to inform the user to include these in his password. Do you still want me to remove it?